libraries/AP_HAL_ChibiOS/hwdef/scripts/defaults_bootloader.h

```diff
@@ -15,11 +15,6 @@
 #define HAL_GCS_ENABLED 0
 #endif
 
-// by default bootloaders don't use INS:
-#ifndef AP_INERTIALSENSOR_ENABLED
```

Comment:
need to test that no bootloader except the CubeNode one changes with this change
so build pre this PR, mv the build to build.orig, re-build PR, compare the bin files with dirdiff or similar to ensure no changes

Reply:
@peterbarker has tested, all good